guide/14-deep-learning/FCN_MLModel_guide-final_draft.ipynb

```diff
@@ -0,0 +1,485 @@
+{
```

Comment:
<p><em>I might change how this comparison is formatted, so it is easier to read the pros and cons of each method. Something like this, maybe?</em></p><p> </p><p>FullyConnectedNetwork | MLModel </p><p> </p><ul><li>Usually requires large dataset| Performs well with comparatively fewer data points </li></ul><p> </p><ul><li>Returns different result for each run due to random weight initialization | More stable than neural network, and usually returns the same result except in some cases which depends on the data used for the model. &nbsp;</li></ul><p> </p><ul><li>It has some universal parameters for fine-tuning, like the number of hidden layers, learning rate, number of epochs, batch size etc | Several  scikit-learn models are available, and each need its own set of unique parameters &amp; hyperparameters setting, followed by fine-tuning. Ideally should be followed by model selection for selecting the optimal model.</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/718/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='45'/>